src/matchers/fail/index.js

```diff
@@ -0,0 +1,10 @@
+import predicate from './predicate';
+
+const failMessage = message => {
+  if (message) return () => message;
+  else return () => 'Failure!';
```

Comment:
Should we have this be something a little bit more descriptive, e.g. 'failed by .fail() assertion.'

Reply:
also it should really be something that is immerative.
Following a similar standard to how git commits should be, (how i like to write test names anyways 😄 
[See this doc](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53#a-properly-formed-git-commit-subject-line-should-always-be-able-to-complete-the-following-sentence)
for example: 'The test ...'

so in this situation you would read 'The test failed by .fail() assertion.'

Reply:
Do you mean imperative mood? Wouldn't it be `fails by .fail() assertion`? I don't think there's a past tense imperative in English, but I'm not a linguist. happy to change it to whatever you'd prefer!

Reply:
i think your right? `fails` make more sense than `failed`, same with `passed` and `passes` too do you think?

Reply:
okay, so I'll change default message to `fails by .fail() assertion` and `passes by .pass() assertion`?

Reply:
yeah, that's perfect 😄

Reply:
updated!